config/jobs/kubernetes/test-infra/janitors.yaml

```diff
@@ -3,17 +3,13 @@ periodics:
   name: maintenance-ci-aws-janitor
   labels:
     preset-service-account: "true"
+  decorate: true
```

Comment:
I'm wondering if the following labels should be used as well:
```
    preset-aws-ssh: "true"
    preset-aws-credential: "true"
```

otherwise, lgtm

Reply:
Thanks, added